exercises/practice/propositional-logic/.docs/instructions.md

```diff
@@ -0,0 +1,31 @@
+# Instructions
+
+Implement a function using Z3's propositional logic capabilities to 
+recreate the following propositional logic properties:
+
+1. A & B == (A' | B')' (DeMorgan's Law)
+2. [A -> (A -> B)] -> (A -> B)
+3. A & (B' -> A') -> B
+
+The function should return the three theorems written in Z3 notation.  The 
+test cases will pass Not(A) and Not(B), Not(A) and B, A and Not(B), and A and 
+B in four separate pairs to the created function.
+
+For reference, the basic propositional logic functions in Z3 are:
```

Comment:
I think you should rewrite this file:

1. Don't explain every step of the code to the user (you're basically writing sudo code for them). They should figure it out. Just write enough info so that the user knows what the purpose of this exercise is.

2. You don't need to continuously mention Z3, as this will be in the Z3 track in Exercism.